scipy/stats/_morestats.py

```diff
@@ -1135,7 +1135,19 @@ def _boxcox_inv_lmbda(x, y):
     return np.real(-num / np.log(x) - 1 / y)
 
 
-def boxcox_normmax(x, brack=None, method='pearsonr', optimizer=None):
+class BigFloat:
+    def __repr__(self):
+        return "BIG_FLOAT"
+
+    def __call__(self, x):
+        dtype = x.dtype if np.issubdtype(x.dtype, np.floating) else np.float64
+        # 10000 is a safety factor because `special.boxcox` overflows prematurely.
+        return np.finfo(dtype).max / 10000
+
+
```

Comment:
I'm sort of regretting my suggestion to add the `__call__` method. It makes the code a bit more opaque because it's not obvious what `ymax = ymax(x)` does. You also have to do:

```python3
dtype = x.dtype if np.issubdtype(x.dtype, np.floating) else np.float64
```

twice this way. So if you are not particularly fond of the `__call__` method, let's get rid of it. Sorry about that.

Reply:
Sure, I've removed the `__call__` method.